ingester-example/src/main/java/io/greptime/QuickStart.java

```diff
@@ -39,7 +38,7 @@ public class QuickStart {
 
     public static void main(String[] args) throws Exception {
         String endpoint = "127.0.0.1:4001";
-        GreptimeOptions opts = GreptimeOptions.newBuilder(endpoint) //
+        GreptimeOptions opts = GreptimeOptions.newBuilder("public", endpoint) //
```

Comment:
consider:

`GreptimeOptions.newBuilder(endpoint, "public")`

Reply:
I will do it on next pr